tensornetwork/tn_keras/test_layer.py

```diff
@@ -397,7 +397,7 @@ def test_config(make_model):
 
   # Serialize model and use config to create new layer
   model_config = model.get_config()
-  layer_config = model_config['layers'][0]['config']
+  layer_config = model_config['layers'][1]['config']
```

Comment:
These seems very sketchy. Why did this change between versions? And is it likely to happen again?

Reply:
`model.get_config()` now returns an `InputLayer` first, while in previous versions the config did not contain an `InputLayer`. If we want to avoid only selecting a specific layer by index, we could loop over layers returned and find the one with `class_name` equal to the one we are interested in. Let me know what you think.